specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -1,11 +1,11 @@
 == [[a235]]Security
 
 This chapter describes the security
-requirements for the Java™ Platform, Enterprise Edition (Java EE) that
-must be satisfied by Java EE products.
+requirements for the Jakarta™ Platform, Enterprise Edition (Jakarta EE) that
```

Comment:
I think we should drop the use of "Platform" in this context since it was referring to the "Java Platform", not the Platform spec.  We should just call this "Jakarta Enterprise Edition".

Reply:
Is Jakarta EE a platform or not?  (And what exactly makes something a "platform"?)
And would we expect other "editions" of the "Jakarta Platform"?

We should be clear on whether we think of this as a platform, but even so that
doesn't require "platform" to be part of the name so I agree with removing it here.